backend/resolvers/queries/portfolioPeriodQuery.js

```diff
@@ -31,3 +36,33 @@ export function portfolioPeriods(_, args, req) {
 
   return PortfolioPeriod.findAll(order);
 }
+
+/**
+ * Just looking for the open portfolio period. If the student requestion (if a student is requesting)
+ * already has a portfolio for that period, return null.
+ */
+export function openPortfolioPeriod(_, args, req) {
```

Comment:
Added this to fetch the open portfolio period before students have added a piece to it.